package.json

```diff
@@ -1,29 +1,27 @@
 {
   "name": "devicepilot",
   "description": "DevicePilot SDK for IoT Analytics",
-  "version": "1.2.5",
-  "main": "index.js",
+  "version": "2.0.0",
+  "main": "./dist/index.umd.js",
   "repository": "https://github.com/DevicePilot/devicepilot-js.git",
-  "homepage": "https://www.devicepilot.com",
-  "author": "thomasmichaelwallace <tom.wallace@devicepilot.com>",
-  "license": "MIT",
   "scripts": {
+    "build": "rollup -c",
```

Comment:
This is the `prepack` script in the npm lifecylce (https://docs.npm.red/misc/scripts)